tests/io/test_from_geopandas.py

```diff
@@ -69,6 +69,7 @@ def test_read_trips_gpd(self):
 
         trips_file = os.path.join("tests", "data", "trips.csv")
         trips_from_csv = ti.read_trips_csv(trips_file, sep=";", tz="utc", index_col="id")
+        trips_from_csv = pd.DataFrame(trips_from_csv)
```

Comment:
I had to add this line to make the test work (since `read_trips_csv` now always returns a GeoDataFrame), but as I commented, the test is rather useless now. Maybe we should add a geojson for `trips.csv`.

Reply:
I am not so sure if I understand what you mean.
Does that mean, that `read_trips_gpd` does not return a geodataframe?

Reply:
Never mind, this changed again with the last commit :)